ethereum/core/src/main/java/org/hyperledger/besu/ethereum/chain/MiningCoordinator.java

```diff
@@ -12,9 +12,8 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-package org.hyperledger.besu.ethereum.blockcreation;
+package org.hyperledger.besu.ethereum.chain;
```

Comment:
Why do we need to move MiningCoordiator to a new package?

Reply:
I had some issues with some classes in some modules. I had to do this to avoid cyclic dependency. But maybe i am doing something wrong. It might even be a bug in my local development environment.

Reply:
It bothers me, but it is turning into a time sync. We need to get the mining coordinator involved to validate the blocks. So I'll put a hold on my original objection in the original bug and we should come back to this later.  Opened [BESU-177](https://jira.hyperledger.org/browse/BESU-177) so we don't forget